docs/adr/0011-create-new-react-app-for-settings-redesign.md

```diff
@@ -0,0 +1,76 @@
+- Pros
+  - Can reuse the existing `fxa-content-server`. At the component level, many just need Mustache/CSS updates.
+  - Requires less initial setup and upfront refactoring so the changes will be in production much more quickly.
+  - Allows us to piecemeal React into Settings, resulting in less future refactoring than option A while avoiding the taboo "ground up remake" approach, and allows for the creation of some React components that could be used elsewhere in the FxA ecosystem.
+- Cons
+  - The full benefits of a React application won't be realized until many more pieces are converted.
+  - Many popular Backbone-to-React integration guides use a combination of React and Backbone with approaches like embedding React components in Backbone views, wrapping the Backbone app with React, and/or keeping data synchronized between a Redux store and Backbone. It's preferable to move away from an interlinked dependency of the two and thus would require refactoring later. Packages (like [NestedReact](https://github.com/VoliJS/NestedReact/blob/master/docs/05_Migration_from_Backbone.md)) offer a convergence layer between the two, but is an additional dependency we would want to remove later.
+  - Integration into `master` must be done carefully and A/B testing these changes may be complicated.
+
+##### Option C - Take the "ground up" approach and create a new settings React application
+
+- Description
+  - Create a new React application for the Settings page and approach it with an incremental view-by-view build up.
+- Pros
+  - Propels us into our new framework of choice and reduces future refactoring efforts.
+  - Settings doesn't have to manage different integrations like the signin/signup flows do. It seems like a reasonable place to begin our conversion regardless of a redesign.
+  - Allows for a fresh start and audit of our HTML semantics, CSS architecture, and accessibility practices as components are being built.
+  - Allows for test auditing and scrutiny. We can potentially remove superfluous tests or add those we lack.
+  - Offers an opportunity to set up and use Storybook in FxA. If we do a view-by-view build up, we can deploy changes somewhere for UX and other folks to see progress and request tweaks as features are built out.
+  - We can potentially use this as an opportunity to review and extract model and API code from the existing settings app into shared modules.
+  - Management in GitHub is easy, as changes can go directly into `master` and no feature flagging or branches held in isolation from `master` will be necessary. A/B testing will also likely be easier.
+- Cons
+  - A "ground up remake" is generally viewed as taboo. Changing such a large chunk of the front-end codebase doesn't come without its own set of risks.
```

Comment:
Yeah, I think we can keep this in mind and mitigate by avoiding a "big bang" surprise replacement. Try to implement and deploy and launch smaller pieces of MVP functionality along the way while keeping the Backbone settings live. Might get tedious at points, but it can be a safety net against losing wisdom earned in the original system